### DISCUSSION TITLE
Over-provision licenses

### DISCUSSION THREAD

**Discussion by northcide** [snipe/snipe-it#8908] [Ideas]
Some licenses are "network" licenses that can be checked in on and as-needed basis and only count cumulatively. It would be great we could have an option to allow for over-provisioning on certain licenses.  Being able to assign 30 users to 15 licenses would be very useful.

Thanks

**Comment by snipe** [comment]
That's also on the list per my reply on the other Discussion issue you had posted. Once we've done that refector, we'll be able to introduce "unlimited" licenses, which will effectively be licenses with no pre-defined limit.
